### PR TITLE
use webauthn instead of webauthn-rp

### PR DESCRIPTION
**Pull request by tkucherera-lenovo**
Constraints: 

1. el9 only support
2.  python 3.8+ support

### CODE REVIEW

File: confluent_server/confluent_server.spec.tmpl [hunk 1]
```diff
@@ -19,15 +19,15 @@ BuildArch: noarch
...
+Requires: python3-pyghmi >= 1.5.71, python3-eventlet, python3-greenlet, python3-pycryptodomex >= 3.4.7, confluent_client == %{version}, python3-pyparsing, python3-paramiko, python3-dns, python3-webauthn, python3-netifaces, python3-pyasn1 >= 0.2.3, python3-pysnmp >= 4.3.4, python3-lxml, python3-eficompressor, python3-setuptools, python3-dateutil, python3-cffi, python3-pyOpenSSL, python3-websocket-client python3-msgpack python3-libarchive-c python3-yaml openssl iproute
 %else
-Requires: python3-dbm,python3-pyghmi >= 1.5.71, python3-eventlet, python3-greenlet, python3-pycryptodome >= 3.4.7, confluent_client == %{version}, python3-pyparsing, python3-paramiko, python3-webauthn-rp, python3-dnspython, python3-netifaces, python3-pyasn1 >= 0.2.3, python3-pysnmp >= 4.3.4, python3-lxml, python3-eficompressor, python3-setuptools, python3-dateutil, python3-cffi, python3-pyOpenSSL, python3-websocket-client python3-msgpack python3-libarchive-c python3-PyYAML openssl iproute
```
Comment (jjohnson42) [inline confluent_server/confluent_server.spec.tmpl:30]: Would probably omit it from the catch all, for now leave it as el9 exclusive

File: confluent_server/confluent/webauthn.py [hunk 2]
```diff
@@ -187,221 +176,107 @@ def update(self, item):
...
+        credential=request,
+        expected_challenge=challenge_model.request,
+        expected_rp_id=APP_RELYING_PARTY.id,
+        expected_origin=APP_ORIGIN,
+        credential_public_key = credential_model.credential_public_key,
+        credential_current_sign_count = 0,
+        require_user_verification = True
+
+    )
+    print(verification)
```
Comment (jjohnson42) [inline confluent_server/confluent/webauthn.py:273]: Might want to remove the devel prints

File: confluent_server/confluent/webauthn.py [hunk 3]
```diff
@@ -187,221 +176,107 @@ def update(self, item):
...
-        APP_CREDENTIALS_BACKEND.handle_credential_assertion(
-            credential=credential,
-            user=user_entity,
-            rp=APP_RELYING_PARTY,
-            expected_challenge=challenge_model.request,
-            expected_origin=APP_ORIGIN
-        )
-    except WebAuthnRPError:
-        raise Exception('Could not handle credential assertion')
+        print(request)
```
Comment (jjohnson42) [inline confluent_server/confluent/webauthn.py:259]: remove devel print

File: confluent_server/VERSION [hunk 4]
```diff
@@ -0,0 +1 @@
+3.11.2~dev56+ga0ffc11d
```
Comment (jjohnson42) [inline confluent_server/VERSION:1]: Don't add the VERSION transient to git

File: confluent_server/LICENSE [hunk 5]
```diff
@@ -0,0 +1,177 @@
+
```
Comment (jjohnson42) [inline confluent_server/LICENSE:1]: Avoid adding the LICENSE from buildrpm to build

File: confluent_server/confluent_server.spec.tmpl (hunk 1 above)
Comment (tkucherera-lenovo) [reply confluent_server/confluent_server.spec.tmpl:30]: new commit with the changes removing the webauthn from else

**Pull request merged**